code/global_vr.dm

```diff
@@ -7,14 +7,14 @@
 	robot_module_types += "Servicehound" //YW changes
 	robot_module_types += "BoozeHound" //YW changes
 	return 1
-
+/*
```

Comment:
//ch removal comment

Reply:
Just realised mobile doesn't quite work well for this. Comment not showing what's below that line.  D'oh, am sleepy